README.md

```diff
@@ -668,6 +684,8 @@ Note that rust version *1.70.0* or later is required.
 
 `make` is also needed for the build.
 
+Make sure to replace **sudo** with **doas** in commands depending on what you use.
```

Comment:
This is also unnecessary. If you use  linux and wen to the effort to install and set up doas, and use that instead of sudo, then you probably already know that you should change the commands to use doas instead of sudo.